packages/fuel-indexer-tests/tests/indexing_postgres.rs

```diff
@@ -595,3 +598,103 @@ async fn test_index_sway_enums() {
     assert_eq!(row.get::<&str, usize>(1), EXPECTED_CONTRACT_ID);
     assert_eq!(row.get::<BigDecimal, usize>(2).to_u64().unwrap(), 0);
 }
+
+#[actix_web::test]
+async fn test_replacing_an_indexer_and_keeping_or_removing_data() {
```

Comment:
@lostman 
- This is a web server test (related to uploading/deploying indexers), not a test related to indexing data (triggering events, indexing the data)
- So we should move this to web_api_postgres.rs (I think that's the name of the module)